File: Anushka-Pachauri/SlidingWindow/MaximumPointsYouCanObtainFromCards.java
class Solution {
    public int maxScore(int[] cardPoints, int k) {
        int score=0;
        for(int i=0;i<k;i++){
            score+=cardPoints[i];
        }
        int maxScore=score;
        for(int i=1;i<=k;i++){
            score-=cardPoints[k-i];
            score+=cardPoints[cardPoints.length-i];
            maxScore=Math.max(maxScore,score);
        }
        return maxScore;
    }
}

File: NikhilKumarGupta/Slidingwindow/Maximum Points You Can Obtain from Cards.cpp
class Solution {
    public:
        int maxScore(vector<int>& cardPoints, int k) {
            int lsum=0;
            int rsum=0;
           int maxsum=0;
          int n=cardPoints.size();
    
            for(int i=0;i<k;i++){
                lsum+=cardPoints[i];
             
            }
               maxsum=lsum;
            int rind=n-1;
            for(int i=k-1;i>=0;i--){
                lsum-=cardPoints[i];
                rsum+=cardPoints[rind];
                rind=rind-1;
                maxsum=max(maxsum,lsum+rsum);
    
    
            }
            return maxsum;
        }
    };